### ISSUE TITLE
test_besttrack_ensemble tests failing 

### ISSUE THREAD

**Issue by WPringle** [noaa-ocs-modeling/EnsemblePerturbation#49]
Hi @zacharyburnettNOAA ,

I merged the latest upstream main into my fork and tried running all the tests. I ran into issues again with the besttrack_ensemble tests (running out of the box after installing):

One of them in the monovariate test (the reference json has everything on just one list but the output json has the curly braces on their own lines):
```
>                   assert len(test_lines) == len(reference_lines), message
E                   AssertionError: "/home/wpringle/EnsemblePerturbation/tests/data/output/test_monovariate_besttrack_ensemble/vortex_1_variable_perturbation_1.json" != "/home/wpringle/EnsemblePerturbation/tests/data/reference/test_monovariate_besttrack_ensemble/vortex_1_variable_perturbation_1.json"
E                     - {
E                     
E                     -   "max_sustained_wind_speed": -1.0
E                     
E                     ? ^^                                ^
E                     
E                     + {"max_sustained_wind_speed": -1.0}
E                     ? ^                                ^
E                     
E                     - }
E                   assert 3 == 1
E                     +3
E                     -1

__init__.py:44: AssertionError
```

The other for the multivariate test doesn't know about "mocker":
```
______________________ ERROR at setup of test_multivariate_besttrack_ensemble ______________________
file /home/wpringle/EnsemblePerturbation/tests/test_track_ensemble.py, line 52
  def test_multivariate_besttrack_ensemble(mocker):
E       fixture 'mocker' not found
>       available fixtures: _socket_marker, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, socket_disabled, socket_enabled, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

```

**Comment by zacharyburnettNOAA** [comment]
ah ok, these are simple fixes, I'll fix these

it looks like I just had the output files on my machine so it was passing the tests automatically

**Comment by zacharyburnettNOAA** [comment]
also I added `pytest-mock` to the requirements, so you might need to `pip install` that as well

**Comment by zacharyburnettNOAA** [comment]
@WPringle fixed with 984703e4d1f9952f4ddc18163165c7596be5c9d1, also you don't need to install `pytest-mock` after all as I didn't actually use it
